domains/authorization/Proxy.IntegrationTests/Setup/ProxyTestAuthHandler.cs

```diff
@@ -21,25 +19,38 @@ protected override Task<AuthenticateResult> HandleAuthenticateAsync()
     {
         var jwtSecurityTokenHandler = new JwtSecurityTokenHandler();
 
-        if (!Context.Request.Headers.ContainsKey(HeaderNames.Authorization))
+        if (!Context.Request.Headers.ContainsKey(HeaderNames.Authorization) ||
+            string.IsNullOrWhiteSpace(Context.Request.Headers[HeaderNames.Authorization]))
```

Comment:
string.IsNullOrWhiteSpace(Context.Request.Headers[HeaderNames.Authorization]

if (!authorizationHeader.StartsWith("Bearer "))

 are both checked by the framework / oidc package. Don't really need to check it twice.